tests/unit/test_providers.py

```diff
@@ -71,7 +73,7 @@ class ProviderWithIncludeFilter(GenericProvider):
 
     # With a non matching data to include, the notification will be skipped and just return empty
     other_fake_data = NotificationData.init_from_raw("other type", b"other data")
-    assert ProviderWithIncludeFilter().get_maintenances(other_fake_data) == []
+    assert not ProviderWithIncludeFilter().get_maintenances(other_fake_data)
```

Comment:
Not sure I see why this is a desirable change - now the test will pass if `get_maintenances()` returns *any* falsey value such as `None`, `0`, etc.

Reply:
I agree on it. I twas a change suggested by the linter as equivalent, but it isn't